plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -1169,6 +1169,28 @@ def ipa_command_param_exists(command, name):
         """
         return api_check_param(command, name)
 
+    def ipa_command_param_has_choice(self, command, name, value):
+        """
+        Check if IPA command support a choice for a specific parameter.
+
+        Parameters
+        ----------
+        command: string
+            The IPA API command to test.
+        name: string
+            The parameter name to check.
+        value: string
+            The parameter value to verify.
+
+        """
+        if name not in api.Command[command].params:
```

Comment:
What about checking if the command exists before checking if the parameter exists?

Reply:
Fixed